modules/assim.batch/R/pda.emulator.R

```diff
@@ -352,22 +369,37 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
 
 
   ## ------------------------------------ Clean up ------------------------------------ ##
-  ## Save outputs to plots, files, and db
-  settings$assim.batch$emulator.path <- file.path(settings$pfts$pft$outdir, 
+  ## Save emulator, outputs files
+  settings$assim.batch$emulator.path <- file.path(settings$outdir, 
```

Comment:
This is fine for now, but lets talk more about the design of the posterior database tables -- the current design suggests you would split up the output file by PFT and possibly even by variable.  We really need to revisit that and either implement it fully or update to a new design that will accommodate partial updating of multiple PFTs in a distributed, hierarchical design.